main.go

```diff
@@ -67,6 +69,9 @@ func main() {
 	flag.BoolVar(&enableServiceMonitors, "with-service-monitors", false, "Enables service monitoring")
 	flag.BoolVar(&enableTLSServiceMonitors, "with-tls-service-monitors", false,
 		"Enables loading of a prometheus service monitor.")
+	flag.BoolVar(&enableLokiStackGateway, "with-lokistack-gateway", false, "Enables lokiStack gateway")
+	flag.BoolVar(&enableTLSLokiStackGateway, "with-tls-lokistack-gateway", false,
+		"Enables loading of a lokiStack gateway.")
```

Comment:
nit. The flag naming is a bit misleading here as per both enabling the lokistack gateway by default. What the flags actually do is:
* `--with-lokistack-gateway`: Enables the manifest creation for the entire lokistack gateway
* `--with-tls-lokistack-gateway`: Configure the lokistack-gateway TLS listener.

I suggest to rename later to `--with-lokistack-gateway-tls-listener`